python/paddle/v2/framework/framework.py

```diff
@@ -313,7 +318,9 @@ def has_var(self, name):
 
     def create_parameter(self, *args, **kwargs):
         global_block = self.program.global_block()
-        return Parameter(global_block, *args, **kwargs)
+        param = Parameter(global_block, *args, **kwargs)
+        self.program.parameters.append(param.name)
```

Comment:
`parameters` is not needed because it is recorded in `vars_`

Reply:
done